change-log.txt

```diff
@@ -1,19 +1,23 @@
 ;;; cl-pdf copyright 2002-2004 Marc Battyani see license.txt for details of the license
 ;;; You can reach me at marc.battyani@fractalconcept.com or marc@battyani.net
 
-This is the change log for cl-pdf. 
+This is the change log for cl-pdf.
 See also contributors.txt and to-do.txt
 
 Current version:
+   This release supports features in the PDF 1.4 specification
```

Comment:
Sometimes, I read text I wrote and get pissed off at myself...

Let's hope nobody expects the word "all" to magically appear.